.github/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ## 0.12.0
 
+1. [EFB/FBW] Added option for TOGA off axis for devices where the TOGA button is built into the throttle - @ImmortalCake
```

Comment:
Please move to the end of the 0.12.0 list.

Reply:
Done